TransferMethod/Sources/UpdateTransferMethodController.swift

```diff
@@ -0,0 +1,473 @@
+    }
+    /// Returns the header view
+    override public func tableView(_ tableView: UITableView,
+                                   willDisplayHeaderView view: UIView,
+                                   forSection section: Int) {
+        guard let headerView = view as?  UITableViewHeaderFooterView
+            else {
+                return
+        }
+        headerView.textLabel?.textColor = Theme.Label.subtitleColor
+    }
+    /// Returns the title for footer
+    override public func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        var footerText = ""
+        if let errorMessage = presenter.sectionData[section].errorMessage {
+            footerText = String(format: "%@", errorMessage)
+        }
+        return footerText
+    }
+    /// Returns the height of header
+    override public func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        if let header = presenter.sectionData[section].header {
+            if header.height(withConstrainedWidth: self.view.frame.width,
+                             font: UIFont.preferredFont(forTextStyle: .body)) > defaultHeaderHeight {
+                return UITableView.automaticDimension
+            } else {
+                return defaultHeaderHeight
+            }
+        } else {
+            return emptyHeaderHeight
+        }
+    }
+    /// Returns height of row
+    override public func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return Theme.Cell.smallHeight
+    }
+    /// Returns tableview section count
+    override public func numberOfSections(in tableView: UITableView) -> Int {
+        return presenter.sectionData.count
+    }
+    /// Returns fields count to update transfer method
+    override public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return presenter.sectionData[section].count
+    }
+    /// Display's the fields to update transfer method
+    override public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let fieldGroup = presenter.sectionData[indexPath.section].fieldGroup
+        let reuseIdentifier = fieldGroup == "UPDATE_BUTTON" ?
+                UpdateTransferMethodButtonCell.reuseIdentifier : UpdateTransferMethodCell.reuseIdentifier
+
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier)
+            else {
+                fatalError("Can't dequeue the cell")
+        }
+        let widget = presenter.sectionData[indexPath.section][indexPath.row]
+        cell.contentView.addSubview(widget)
+        if let widget = widget as? AbstractWidget, widget.errorMessage != nil {
+            widget.showError()
+        }
+
+        if fieldGroup == "INFORMATION" {
+            cell.backgroundColor = Theme.Cell.disabledBackgroundColor
```

Comment:
there's no `fieldGroup == INFORMATION` in Update. This section contains Fees and Processing time that's only displayed in Add Transfer method. So you can delete all the code related to `fieldGroup == INFORMATION` from Update classes

Reply:
Done